### PR TITLE
Initial commit for Simple Session Tracking Lua script (used with tslu…

### PR DESCRIPTION
**Pull request by sandmanahoy**
…a plugin)

### CODE REVIEW

File: traffic_server/plugins/sst/sst.lua [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+--  See the License for the specific language governing permissions and
+--  limitations under the License.
+
+package.path = package.path .. ";/opt/trafficserver/script/uuid.lua"
+local uuid = require("uuid")
+local QUERY_STRING_PARAMS = ""
+
+function __init__(argtb)
+    local unique_id = tostring( {} ):sub(8) -- memory address of blank table
+    local x = unique_id + os.time() -- plus current time
```
Comment (shukitchan) [inline traffic_server/plugins/sst/sst.lua]: you can use ts.now() instead here 
https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/ts_lua.en.html#ts-now

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/21/

**Comment by mitchell852** [comment]
@sandmanahoy - let me know if this was tested and if this is something you still want merged.

**Pull request closed without merging**